### ISSUE TITLE
xcode9 iOS 11 - Stereo Mode is not supported

### ISSUE THREAD

**Issue by ElizaSapir** [ejeinc/MetalScope#28]
Hey,

I am getting the following issue when running StereoVideo example and trying to move to stereo mode:

```
validateRenderPassDescriptor:487: failed assertion `Texture at colorAttachment[0] has usage (0x01) which doesn't specify MTLTextureUsageRenderTarget (0x04)'
```

<img width="1015" alt="screen shot 2017-09-20 at 4 52 37" src="https://user-images.githubusercontent.com/3921852/30623278-4ed34a8e-9dbf-11e7-8fb6-7a044d9f7644.png">

Can you help here? it worked like a charm on iOS10 when moved to iOS11 got this issue.

**Important:**
When I disable Metal API validation the above issue is ignored, BUT stereo mode shows a black screen with no split view.

**More Details:**

When replacing under `StereoRenderer` class `eyeTextureDescriptor` instance `usage` property to `.unknown` there is no crash issue but still get black screen. 

```
eyeTextureDescriptor.usage = .unknown
```

Thanks!

**Comment by giladna** [comment]
+1

**Comment by KaushikPulagara** [comment]
+1. Stereo mode doesnt work in ios 11. View is just black.

**Comment by Allui** [comment]
eyeTextureDescriptor.usage = .unknown
+
var panoramaView: PanoramaView?  (**remove weak**)

**Comment by junpluse** [comment]
I'm sorry to have kept you waiting. I'm currently working on this problem, so please wait for a while 🙇

**Comment by ElizaSapir** [comment]
Hey @junpluse,

Thanks for your response, do you have any estimation when it can be resolved? It's really important for us since we uploaded an app to Appstore and now we have this issue :(

BR,
Eliza

**Comment by junpluse** [comment]
Hi @ElizaSapir,

Thank you for reporting the issue.
I just published `0.16.0` including quick fixes for this problem.
https://github.com/ejeinc/MetalScope/releases/tag/0.16.0

Please check it out.

**Comment by ElizaSapir** [comment]
@junpluse,

Thank you very much for your quick response!
Tested on my side and now it looks great!

Best,
Eliza
